README.md

```diff
@@ -1 +1,113 @@
-# school-library
\ No newline at end of file
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+- [💻 Getting Started](#getting-started)
+  - [Prerequisites](#prerequisites)
+  - [Installation](#installation)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ](#faq)
+- [📝 License](#license)
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 School library <a name="about-project"></a>
+
+**School library** management system is a comprehensive software project designed to streamline the management of students, teachers, and administrative tasks within an educational institution. This multi-phase project consists of seven key components, each aimed at enhancing functionality, maintainability, and usability.
+
+## 🛠 Built With <a name="built-with"></a>
+
+### Tech Stack
+
+This project is built using the following technologies:
+
+<details>
+<summary>Programming Languages</summary>
+  <ul>
+    <li>Ruby</li>
+  </ul>
+</details>
+
+### Key Features <a name="key-features"></a>
+
+<details>
+  <ul>
+    <li>Creation and management of student and teacher profiles.</li>
+    <li>Authentication and authorization to restrict access to authorized users.</li>
+  </ul>
+</details>
+
+
+<!-- GETTING STARTED -->
+
+## Getting Started <a name="getting-started"></a>
+
+To get started with the School library, follow the steps below:
+
+### Prerequisites
+
+Before running this project, make sure you have the following prerequisites installed:
+
+- Ruby
+
+### Installation <a name="installation"></a>
+
+1. Clone this repository to your desired folder:
+
+   ```sh
+   git clone https://github.com/Ochiengsteven/school-library.git
+
+2. Navigate to the project directory:
+
+    ```sh
+    cd school_library
+3. Run the program:
+    ```sh
+    ruby school_library.rb
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+👤 Ochieng Steven
+
+GitHub: @Ochiengsteven<br>
+Twitter: @ochiengotieno63<br>
+
+## 🔭 Future Features <a name="future-features"></a>
+
+<details>
+<summary>In near future I am planning to implement parent portal</summary>
+  <ul>
+    <li>Mobile app for parents to access real-time information about their child's performance, attendance, and school activities.</li>
+    <li>Allow parents to communicate with teachers and administrators, schedule parent-teacher conferences, and receive notifications about important events or announcements.</li>
+    <li>Provide a secure and convenient way for parents to stay actively involved in their child's education.</li>
+  </ul>
+</details>
```

Comment:
- Great job adding all of these that you'd implement in the future, however, following the project milestones, none of the aforementioned points are stated in the remaining parts of the project milestone to completing this project. Please, refer to [this link](https://github.com/microverseinc/curriculum-ruby/blob/main/oop/sneak_peek.md) to see other parts of the project milestones that would be required of you to implement. Add them to your future section as they are relevant to this project to make your project more consistent